unfetter-discover/unfetter-ui
# Issue 956 (dynamic steps for categories in baselines wizard)

Fixes unfetter-discover/unfetter#956.

This introduces first step of baselines wizard.  Some logic is still needed to transition between the  categories selection, categories pages, and summary, but this gets it going well.  User should be prevented from adding two of the same category, and currently user has to click Add to ensure it is marked for inclusion in baseline.  Next PR will not require Add button click.
 
## To test:
1. Pull this PR
2. Restart UI:   `docker-compose restart unfetter-ui`
3. In Assessments 3.0, show master list of baselines and click Create
4. Enter title, description, and org and click Continue
5. Verify that first step of wizard appears, where you can select a category, click the Add icon, and repeat for other categories
6. Verify that clicking on Continue introduces added categories as new steps in the wizard (between 'Categories' and 'Summary')

Note:  icons for add/delete are not final, and spacing for sidebar is not wide enough yet.  Will address in next PR.
